scipy/constants/codata.py

```diff
@@ -1139,12 +1141,365 @@
 Wien frequency displacement law constant               5.878 9238 e10        0.000 0034 e10        Hz K^-1
 Wien wavelength displacement law constant              2.897 7729 e-3        0.000 0017 e-3        m K"""
 
+txt2018 = """\
+alpha particle-electron mass ratio                          7294.299 541 42          0.000 000 24             
+alpha particle mass                                         6.644 657 3357 e-27      0.000 000 0020 e-27      kg
+alpha particle mass energy equivalent                       5.971 920 1914 e-10      0.000 000 0018 e-10      J
+alpha particle mass energy equivalent in MeV                3727.379 4066            0.000 0011               MeV
+alpha particle mass in u                                    4.001 506 179 127        0.000 000 000 063        u
+alpha particle molar mass                                   4.001 506 179 127 e-3    0.000 000 000 063 e-3    kg mol^-1
```

Comment:
Looks like the units are getting clipped.

Reply:
What gives you the impression? The lines shown above are identical to the text file from nist.

Reply:
Oops, I didn't see the slider under the text, so it looked truncated.